### PR TITLE
Research articles

### LINKED ISSUE #61: Research Page - Ongoing Projects
image

### PR DESCRIPTION
**Pull request by LemonFace0309**
closed #61 #62 

image

image

I also noticed that many of our pages break at smaller viewports. This is primarily due to the the nav bar taking up more space that the device's width. Just something to keep in mind to save future frustration.

### CODE REVIEW

**Comment by functionstackx** [comment]
@LemonFace0309 can you push a  dumby commit here, format action is wacky rn LOL,


it passed and it applied the changes, but didn't change the check to pass

https://github.com/Cache-and-Cookies/msmr/actions/runs/599248172

File: components/Research/OngoingProjects.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <Col xs={12} className="text-center">
+          <h2>Ongoing Projects</h2>
+          <h6 className="pb-4">
+            The following projects are using the MSMR and MSMR-I in the capacity
+            to assess and track resilience over time:
+          </h6>
+        </Col>
+      </Row>
+      <Row>
+        <Col xs={12} md={3} className={styles.col}>
```
Comment (jimmyygit) [inline components/Research/OngoingProjects.js]: How come these `Col` components have different classNames?

Also.
image

Ongoing Projects to Publications have different alignents on smaller viewports, but you can worry about this later.

**Review by jimmyygit — APPROVED** [review]
Some responsiveness issues but look really good :)

File: components/Research/OngoingProjects.js (hunk 1 above)
Comment (LemonFace0309) [reply components/Research/OngoingProjects.js]: hehe mb i was testing something out

**Pull request merged**